tools/avatar-generator-GUI/README.md

```diff
@@ -0,0 +1,49 @@
+# avatar-generator-GUI
```

Comment:
I would suggest placing all the png files into a daughter dir, to make the code base easier to navigate